database/src/main/kotlin/no/nav/su/se/bakover/database/beregning/PersistertMerknad.kt

```diff
@@ -0,0 +1,128 @@
+package no.nav.su.se.bakover.database.beregning
+
+import com.fasterxml.jackson.annotation.JsonSubTypes
+import com.fasterxml.jackson.annotation.JsonTypeInfo
+import no.nav.su.se.bakover.domain.beregning.Merknad
+import java.time.LocalDate
+
+internal sealed class PersistertMerknad {
```

Comment:
Ser ikke ut som noe extender denne direkte? Brukes den kun for "namespace"?

Reply:
Først og fremst "namespace", men tanken var vel at beregning skulle extende (glemt å legge til/har ikke noe særlig betydning for øyeblikket)